backend/app/migrations/env.py

```
@@ -0,0 +1,85 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+
+from dotenv import load_dotenv
+import os
+load_dotenv()
+connectable = create_engine(os.getenv("DATABASE_URL"))
```

Ruff (0.8.2)

Undefined name `create_engine`

(F821)